### PR TITLE
MYST-413 Optimize dockers to have more cached layers

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

**Comment by tadovas** [comment]
What about alpine image building? Needs to be migrated to our builder too?

**Comment by Waldz** [comment]
Had same idea, but alpine currently compiled in alpine, and builder is Ubuntu image

File: bin/client_package/config/update-resolv-conf [hunk 1]
```diff
@@ -20,7 +17,11 @@ NMSRVRS=()
 SRCHS=()
 adapters=()
 
+# TODO Investigate DNS on Linux environment
 NETWORKSETUP=/usr/sbin/networksetup
+if [ ! -x "$NETWORKSETUP" ]; then
```
Comment (tadovas) [inline bin/client_package/config/update-resolv-conf]: NETWORKSETUP=(which networksetup)
if [ - "$NETWORKSETUP" ]; then //not OSX
don't rely on hardcoded path - maybe we decide to bundle our own

Comment (Waldz) [reply bin/client_package/config/update-resolv-conf]: Removed my logic, waiting for your fix

**Pull request merged**